.github/workflows/test.yml

```diff
@@ -142,3 +142,17 @@ jobs:
       with:
         name: Test Reports
         path: build/reports/tests
+
+    - name: "Notify QA Slack"
+      if: steps.set-env-step.outputs.test-env == 'alpha' || steps.set-env-step.outputs.test-env == 'staging'
+      uses: broadinstitute/action-slack@v3.8.0
+      # see https://github.com/broadinstitute/action-slack
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we still need this, or is it just left over from the old version? See https://action-slack.netlify.app/migration/01-migrate-from-v2-to-v3

Reply:
@zarsky-broad good call, removed `GITHUB_TOKEN`